readme.rst

```diff
@@ -25,84 +25,90 @@ Prerequisites
 Installation
 ~~~~~~~~~~~~
 * Clone the repository:
-::
+  ::
 
-  # from within your preferred source development directory
-  git clone https://github.com/ImageMarkup/isic-archive.git
-  cd isic-archive
-  git submodule init && git submodule update
+    # from within your preferred source development directory
+    git clone https://github.com/ImageMarkup/isic-archive.git
+    cd isic-archive
+    git submodule init && git submodule update
```

Comment:
If we want, this can be shortened to the equivalent `git submodule update --init`.